gap/oper.gi

```diff
@@ -193,9 +193,8 @@ end);
 InstallMethod(DigraphAddEdge,
 "for an immutable digraph and two positive integers",
 [IsImmutableDigraph, IsPosInt, IsPosInt],
-function(D, src, ran)
-  return MakeImmutable(DigraphAddEdge(DigraphMutableCopy(D), src, ran));
-end);
+{D, src, ran}
```

Comment:
I'm not sure why these changes are in this PR, it looks like you might have had a slightly older version of Digraphs than the most up to date. Can you please rebase your changes onto the current main branch?